src/streaming/extensions/ProtectionExtensions.js

```diff
@@ -237,6 +237,8 @@ MediaPlayer.dependencies.ProtectionExtensions.prototype = {
                     protCtrl.selectKeySystem(keySystemAccess);
                 } else {
                     self.log(event.error);
+					protCtrl.notify(MediaPlayer.dependencies.ProtectionController.eventList.ENAME_PROTECTION_ERROR,
+                        "[DRM] KeySystem Access Denied! -- " + event.error);
                 }
```

Comment:
@nicosang Could you update this to use spaces instead of tabs for indent?  Thanks for the fix!